README.md

```diff
@@ -68,6 +68,11 @@ jobs:
   build:
     # Call this shared workflow (use `@v1` to use a specific version, instead of the latest)
     uses: sentrysoftware/workflows/.github/workflows/maven-build.yml@main
+    permissions:
+      packages: write
```

Comment:
Are these permissions really necessary? I ran it in the Jawk project without these permissions, and it worked (all comments were posted to the PR).

Reply:
My bad, I will remove this.